source/common/http/http1/codec_impl.cc

```diff
@@ -21,6 +21,12 @@ namespace Http1 {
 const std::string StreamEncoderImpl::CRLF = "\r\n";
 const std::string StreamEncoderImpl::LAST_CHUNK = "0\r\n\r\n";
 
+StreamEncoderImpl::StreamEncoderImpl(ConnectionImpl& connection) : connection_(connection) {
```

Comment:
Same question as above about when this call happens in relation to the `newStream()` call.